relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -331,6 +334,17 @@ pub(crate) fn extract_tags(
             span_tags.insert(SpanTagKey::Group, span_group);
 
             let truncated = truncate_string(scrubbed_desc, config.max_tag_value_size);
+            if span_op.starts_with("resource.") {
```

Comment:
Should we move this to the below `span_op.starts_with("resource.")` block, so all the resource extraction is one place?